packages/utils/recursive-parser.test.js

```diff
@@ -21,7 +21,61 @@ defineTest(
 			}
 		],
 		nice: "':)'",
+		foo: "Promise.resolve()",
 		man: "() => duper"
 	},
 	"init"
 );
+
+// Tests for _add_ command
+defineTest(
```

Comment:
This is nice. Just have one test case with a lot of properties, that will hold and tests would run faster

Reply:
No need to comment this, the action specifies what the test is supposed to do